tests/unittests/tests-printf_float/tests-printf_float.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup  unittests
+ * @{
+ *
+ * @file
+ * @brief       Implementations of unit tests for printing floating point numbers
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @}
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "embUnit/embUnit.h"
+
+#include "tests-printf_float.h"
+
+#define BUFSIZE             (10)
+
+static const float in0 = 2016.0349f;
+static const float in1 = 123.4567f;
+static const float in2 = 0.0f;
+
+static void sfprintf_float(void)
+{
+    char tmp[BUFSIZE];
+    char *str = tmp;
```

Comment:
As in the original PR, I'm asking for the sense of the intermediate variable `tmp`. Wouldn't `char str[BUFSIZE];` serve the same purpose (and spare a small amount of memory)?